specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/environmentContainer.json

```diff
@@ -0,0 +1,156 @@
+      "get": {
+        "tags": [
+          "EnvironmentContainer"
+        ],
+        "operationId": "EnvironmentContainers_List",
+        "x-ms-examples": {
+          "List Environment Specifications.": {
+            "$ref": "./examples/listEnvironmentContainers.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/PaginationParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/EnvironmentContainerResourceArmPaginatedResult"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {},
+  "definitions": {
+    "EnvironmentContainer": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The asset container description text.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "The asset tag dictionary. Tags can be added, removed, and updated.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "latestVersions": {
+          "description": "Latest Versions for all assets",
+          "type": "array",
+          "items": {
+            "$ref": "environmentSpecification.json#/definitions/EnvironmentSpecificationVersionResource"
+          }
+        }
+      }
+    },
+    "EnvironmentContainerResource": {
+      "description": "Azure Resource Manager resource Envelope",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource URL of the entity (not URL encoded).",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource provider and type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/EnvironmentContainer"
```

Comment:
Could we rename this model to `EnvironmentContainerProperties` to emphasize this is only the properties rather than the resource itself?

Reply:
this is what we're following for all asset types.
See https://github.com/Azure/azure-rest-api-specs/pull/10876